docs/markdown/snippets/introspect_breaking_format.md

```diff
@@ -0,0 +1,11 @@
+## Changed the JSON format of the introspection
+
+All paths used in the meson introspection JSON format are now absolute. This
+affects the `filename` key in the targets introspection and the output of
+`--buildsystem-files`.
+
+Furthermore, the `filename` and `install_filename` keys in the targets
+introspection are now lists of strings with identical length.
+
+The `--traget-files` option is now deprecated, since the same information
```

Comment:
This should be `--target-files`

Reply:
OH... Thanks for catching that. Spelling is hard :)